drivers/SmartThings/matter-sensor/src/test/test_matter_sensor.lua

```diff
@@ -18,6 +18,8 @@ local capabilities = require "st.capabilities"
 local t_utils = require "integration_test.utils"
 local clusters = require "st.matter.clusters"
 
+test.set_rpc_version(4)
```

Comment:
Can you add a test case that checks that nothing happens when RPC version < 4?

Reply:
Added a new test case in 4c9707b7c23b9fbbfe370a4ab591bc9c68774041.